documentation/long-stay-baseline.mdx

```diff
@@ -0,0 +1,528 @@
+
+The approach involved:
+
+1. Defining the population and data cleaning
+2. Feature engineering, focussing on basic numerical and categorical features
+3. Simple baseline models implemented using commonly available packages including [scikit-learn 1.1.1](https://scikit-learn.org/), [CatBoost 1.0.6](https://catboost.ai) and [XGBoost 1.3.3](https://xgboost.readthedocs.io/en/stable/)
+4. Analysis of model performance by demographic
+5. A comparison of regression-based and classification-based risk stratification models
+6. A set of extensions for future work
+
+## 3. Data ingest and processing<a name='dataingestandprocessing'></a>
+
+GHNHSFT performed a SQL export from their EPR system containing ~770,000 records across 99 columns, with significant sparsity across several columns and a section of rows, as visualised by light coloured blocks (null values) in the below image:
+
+![Image of data sparsity](images/long-stay-baseline/sparsity.jpeg)
+
+The population for this study was defined as non-elective, major cases as recorded in the `IS_MAJOR` and `elective_or_non_elective` fields.
+
+Filtering the dataset to this definition resulted in a reduction of ~770,000 rows to ~170,000 rows (78% reduction):
+
+![Image of data sparsity - major cases](images/long-stay-baseline/sparsity-major.jpeg)
+
+Data was processed by:
+
+1. Converting datetime columns into the correct data type
+2. Ordering records by `START_DATE_TIME_HOSPITAL_PROVIDER_SPELL`
+3. Removing fields not available at admission
+4. Removing empty and redundant (e.g. `LENGTH_OF_STAY_IN_MINUTES` duplicates `LENGTH_OF_STAY`) columns
+5. Removing duplicate rows
+4. Removing local identifiers
+5. Imputing `stroke_ward_stay` as `N` if not specified
+6. Binary encoding `stroke_ward_stay`, `IS_care_home_on_admission`, `IS_care_home_on_discharge` and `IS_illness_not_injury`
+
+This resulted in ~170,000 rows across ~50 columns as visualised in the below image:
+
+![Image of data sparsity of "clean" data](images/long-stay-baseline/sparsity-clean.jpeg)
+
+The resulting data dictionary is available [here](https://github.com/nhsx/skunkworks-long-stayer-risk-stratification-baseline/blob/main/docs/data-dictionary.csv).
+
+Additionally, Length of Stay was capped to 30 days, due to a long tail of long stayers over ~15 days and the definition of long stayer being over 21 days. The effect of capping can be visualised by comparing box plots of the distribution of length of stay on the raw data (left image, y scale up to 300 days) and the capped data (right image, y scale up to 30 days):
+
+<p align="center">![Boxplot of length of stay](images/long-stay-baseline/los-boxplot.png)</p>
+
+The resulting distribution of length of stays shows a ~bimodal distribution - the majority of stays are short (&lt;5 days), which a long tail and population of long stayers:
+
+<p align="center">![Density plot of length of stay](images/long-stay-baseline/los-density.jpeg)</p>
```

Comment:
The peak for the long stayer (>30 days):
- Worth mentioning the bimodal behaviour is caused by grouping all >30 days examples to 1 bin.
- x-axis ticks: should show > 30 days for that bin, instead of just 30 days.